test/benchmarks/run.sh

```diff
@@ -28,33 +28,55 @@ SOLIDITY_BUILD_DIR=${SOLIDITY_BUILD_DIR:-${REPO_ROOT}/build}
 
 result_legacy_file=$(mktemp -t benchmark-legacy-XXXXXX.txt)
 result_via_ir_file=$(mktemp -t benchmark-via-ir-XXXXXX.txt)
+warnings_and_errors_file=$(mktemp -t benchmark-warn-err-XXXXXX.txt)
```

Comment:
With this many files I'd recommend to just switch to a dir instead:
```suggestion
output_dir=$(mktemp -d -t solc-benchmark-XXXXXX)
```

And then you can just use arbitrary file names inside: `${output_dir}/legacy.txt`, `${output_dir}/via-ir.txt` and `${output_dir}/warnings_and_errors.txt`.

Less litter in `/tmp` and cleanup is simple (just remove the dir).